fix snapcraft build

```
Run VERSION=`snapcraft list-revisions slcli --arch armhf | grep "edge\*" | awk '{print $1}'`
craft-store error: No keyring found to store or retrieve credentials from.     
Recommended resolution: Ensure the keyring is working or SNAPCRAFT_STORE_CREDENTIALS is correctly exported into the environment                                
For more information, check out: https://snapcraft.io/docs/snapcraft-authentication
Full execution log: '/home/runner/.cache/snapcraft/log/snapcraft-20220[6](https://github.com/softlayer/softlayer-python/runs/7138330636?check_suite_focus=true#step:3:7)30-192939.[8](https://github.com/softlayer/softlayer-python/runs/7138330636?check_suite_focus=true#step:3:9)6588[9](https://github.com/softlayer/softlayer-python/runs/7138330636?check_suite_focus=true#step:3:10).log'                                                                   
Publishing on armhf
Usage: snapcraft [OPTIONS] COMMAND [ARGS]...
Try 'snapcraft -h' for help.
Error: No such command 'release'.
Error: Process completed with exit code 2.

```

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -46,6 +46,7 @@ jobs:
     - name: Publish 📦 to Test PyPI
       uses: pypa/gh-action-pypi-publish@master
       with:
-        password: ${{ secrets.CGALLO_TEST_PYPI }}
-        repository_url: https://test.pypi.org/legacy/
+        user: __token__
+        password: ${{ secrets.CGALLO_PYPI }}
+        repository_url: https://pypi.org/legacy/
 

